### PR TITLE
Fix CI Full to run both 1.12 and 1.13

### PR DESCRIPTION
**Pull request by jakubtobiasz**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.13                   |
| Bug fix?        | no                                                       |
| New feature?    | no                                                       |
| BC breaks?      | no                                                       |
| Deprecations?   | no  |
| Related tickets | n/a                      |
| License         | MIT                                                          |

It should run both 1.12 and 1.13 now.

### CODE REVIEW

**Comment by dawkaa** [comment]
Hi dude, how to test the CI on the local environment?

**Comment by jakubtobiasz** [comment]
@dawkaa my latest PRs are aiming to provide complete CI for GHA. Maybe some day I'll try to provide better experience locally, but for now, you have to run each command manually.

File: .github/workflows/ci_e2e-custom.yaml [hunk 1]
```diff
@@ -21,7 +27,14 @@ jobs:
             DATABASE_URL: "mysql://root:root@127.0.0.1/sylius?charset=utf8mb4&serverVersion=${{ matrix.mysql }}"
 
         steps:
-            -
+            -   name: "Checkout (With Branch)"
```
Comment (GSadee) [inline .github/workflows/ci_e2e-custom.yaml:30]:
```suggestion
            -   name: "Checkout (with branch)"
```
?

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, Jakub! :tada:

**Pull request merged**